src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,54 @@
+import { memo, useState } from 'react';
+
+import './dropdown.scss';
+import '../../styles/buttons.scss';
+
+type DropdownProps = { options: string[] };
+
+const Dropdown = ({ options }: DropdownProps) => {
+  const [selected, setSelected] = useState(options[0]);
```

Comment:
Most of the time you will need to forward initial value which is optional
```suggestion
  const [selected, setSelected] = useState(initialValue);
```